Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -180,9 +180,11 @@ public override Ust VisitAssignmentExpression(AssignmentExpressionSyntax node)
             }
             else
             {
-                var op = BinaryOperatorLiteral.TextBinaryOperator[opText.Remove(opText.Length - 1)];
+                BinaryOperator op = BinaryOperatorLiteral.TextBinaryOperator[opText.Remove(opText.Length - 1)];
                 // TODO: implement assignment + operator
                 result = new AssignmentExpression(left, right, node.GetTextSpan());
+                result.BinaryOperator = new BinaryOperatorLiteral(
+                        op, node.OperatorToken.GetTextSpan());
```

Comment:
It's better to use [Object Initializer](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers) syntax here.